### DISCUSSION TITLE
After updating I can't use the scripts, None of them

### DISCUSSION THREAD

**Discussion by Snoopo69** [thatskymod/Sky-CotL-Scripts#69] [General]
The magic is gone since I updated the game

**Comment by thatskymod** [comment]
## Resolved in Dukun Langit v0.2.3rc

The issue shall now be revolved as of **Dukun Langit v0.2.3rc**.

The file is available for download down below:

https://github.com/thatskymod/Sky-CotL-Scripts/blob/d021c69403acb5cc6f9d705c510bc3f110e47bb7/Canvas/Libs/libLangit023rc.so

**Note for future readers:**
With the rapid pace at which mods are evolving, it is difficult to keep links on aged discussions up-to-date. If you find that the link is broken, consider searching the repository manually.
